images/user/environment.yml

```diff
@@ -31,17 +31,15 @@ dependencies:
 
   - pip:
     # Infrastructure things
-    - jupyterlab==2.*
+    - jupyterlab==3.*
     - voila==0.1.23
     - nbgitpuller==0.9.*
     - jupyter-resource-usage==0.5.0
     - nbinteract==0.2.5
-    - jupytext==1.6.*
     - RISE==5.6.1
     - jupyter_contrib_nbextensions==0.5.1
     - notebook==6.1.4
     - jupyterhub==1.1.0
```

Comment:
While it may be off topic, I think bumping jupyterhub to 1.3 should be done if we have z2jh 0.11 at least.